lambda/simplif.ml

```diff
@@ -220,6 +220,7 @@ let simplify_exits lam =
   | Lprim(p, ll, loc) -> begin
     let ll = List.map simplif ll in
     match p, ll with
+    (* FIXME does this respect Rc_nontail? *)
```

Comment:
I think this is safe as it is, but `(x |> foo y) [@nontail]` will compile as `(foo y) x` rather than `foo y x`.